src/pages/api/breaches.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import axios from 'axios';
+import { HaveIbeenPwnedText } from '@/assets/types/have-i-been-pawned';
+
+interface BreachesProps {
+  textContent: HaveIbeenPwnedText['HeroSection']['breaches'];
+}
+
+const API_URL = process.env.HIBP_API_URL;
+const API_KEY = process.env.HIBP_API_KEY;
+
+const cache: Map<string, any> = new Map();
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+  textContent: BreachesProps['textContent'],
+): Promise<void> {
+  if (req.method !== 'GET') {
+    res.status(405).json({ error: textContent.error405 });
+    return;
+  }
+  const CACHE_CLEAN_INTERVAL_MS = 2 * 60 * 60 * 1000;
```

Comment:
Extract this constant outside the function.

Reply:
Done